src/map/attack.cpp

```diff
@@ -375,10 +384,10 @@ bool CAttack::CheckAnticipated()
         {
             //increment power and don't remove
             effect->SetPower(effect->GetPower() + 1);
-            //chance to counter - 25% base
-            if (tpzrand::GetRandomNumber(100) < 25 + m_victim->getMod(Mod::THIRD_EYE_COUNTER_RATE))
+            //chance to counter - 15% base
```

Comment:
Did end up deciding that the change was a nerf rather than a buff? I think I remember coming to the conclusion that it was a flat 25% before the buffly worded the nerf.